### DISCUSSION TITLE
[Bug] so what is the default passphrase?

### DISCUSSION THREAD

**Discussion by jonny77** [Chia-Network/chia-blockchain#9721] [Support]
### What happened?

I remember that I did not set any passphrase before.
<img width="368" alt="WX20211221-104035@2x" src="https://user-images.githubusercontent.com/1962170/146861776-cff681d4-3d9d-419b-8049-b6523fe4757c.png">

### Version

latest

### What platform are you using?

Windows

### What ui mode are you using?

GUI

**Comment by Mattjp365** [comment]
I have the same issue same OS

**Comment by emlowe** [comment]
There is no default passphrase, you must have entered one at one point.
